.github/generate-envs.py

```diff
@@ -191,7 +191,7 @@
     {
         "lang": "verilog",
         "sim": "verilator",
-        "sim-version": "v5.036",  # Latest release version.
+        "sim-version": "v5.038",  # Latest release version.
```

Comment:
I guess this is the first time in a long time that we have more than one supported Verilator version? We need the older versions in extended CI then, just like we do for Questa & friends.